science/projects/Ghazaleh Mahmoudi/01-main-content/main_content_detection.ipynb

```diff
@@ -0,0 +1,1400 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.                except:
</pre><p>Generally speaking, abusing try/except is not considered a good practice, although adding documentation to the except block is a good practice to add code readability to the code </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>